docker-compose.yml

```diff
@@ -30,3 +30,14 @@ services:
       - django
     ports:
       - "80:80"
+
+  render:
+    build:
```

Comment:
Dockerfile should be explicitly stated here (but you might not be using Compose yet)